symex-interface-clojure.el

```diff
@@ -75,6 +75,20 @@ Accounts for different point location in evil vs Emacs mode."
   "Evaluate buffer."
   (cider-eval-buffer))
 
+(defun symex-clojure-comment-remaining-reader-macro ()
+  "Comment out remaining symexes at this level using clojure's comment reader macro."
+  (interactive)
+  (let ((count (symex--remaining-length)))
+    (symex-clojure-comment-reader-macro count)))
+
+(defun symex-clojure-comment-reader-macro (count)
```

Comment:
I don't see these functions being used anywhere - is the plan for them to be used from [`symex-comment`](https://github.com/countvajhula/symex.el/blob/ce33a7858481da2a1a07558b0932b19328d3449b/symex-transformations.el#L577-L587) and `symex-comment-remaining`?